docs/sql-data-sources-jdbc.md

```diff
@@ -213,6 +213,14 @@ the following case-insensitive options:
   </tr>
 </table>
 
+It worth to mention that kerberos authentication with keytab is not always supported by the JDBC driver.<br>
+Before using <code>keytab</code> and <code>principal</code> configuration options please make sure the following requirements met:
+<ol>
+   <li> The included JDBC driver version supports kerberos authentication with keytab. </li>
```

Comment:
Could you use Markdown notations instead of HTML tags?
```
 * The included JDBC driver version supports kerberos authentication with keytab. 
 * ...
```

Reply:
Changed.